xml/xml-quiz.md

```diff
@@ -150,5 +150,54 @@ XML Assessment
 - Set the maxOccurs attribute to undefined.
 - Set the maxOccurs attribute to unbounded.
 
+<<<<<<< Updated upstream
+=======
```

Comment:
this line shows you that there are some conflicts inside this branch, please fix it before I will merge it